loadgen/cmd/otelbench/flags.go

```diff
@@ -133,8 +134,17 @@ func Init() error {
 
 	// `concurrency` is similar to `agents` config in apmbench
 	// Each value passed into `concurrency` list will be used as loadgenreceiver `concurrency` config
-	flag.Func("concurrency", "comma-separated `list` of concurrency (number of simulated agents) to run each benchmark with",
+	flag.Func("concurrency", "comma-separated `list` of concurrency (number of simulated agents) to run each benchmark with. Use 'auto' to set concurrency based on GOMAXPROCS",
 		func(input string) error {
+			// Handle 'auto' value specially
+			if input == "auto" {
+				// Use the current GOMAXPROCS value
+				n := runtime.GOMAXPROCS(0)
+				fmt.Printf("using GOMAXPROCS value %d for concurrency\n", n)
```

Comment:
```suggestion
				fmt.Fprintf(os.Stderr, "using GOMAXPROCS value %d for concurrency\n", n)
```

iirc it needs to go to stderr to avoid being parsed as part of benchmark output